arbitrum-docs/launch-orbit-chain/how-tos/arbos-upgrade.md

```diff
@@ -10,23 +10,33 @@ sidebar_position: 3
 
 This how-to provides step-by-step instructions for Orbit chain operators who want to upgrade ArbOS on their Orbit chain(s). Familiarity with ArbOS, Orbit, and [chain ownership](../concepts/chain-ownership.md) is expected. Note that Orbit chain owners have full discretion over when and whether to upgrade their ArbOS version.
 
+The specific upgrade requirements for each ArbOS release are located under each reference page for that specific [ArbOS release](../../node-running/reference/arbos-software-releases/overview.mdx#list-of-available-arbos-releases).
+
 import PublicPreviewBannerPartial from '../../node-running/partials/_upgrade-cadence-recommendations-partial.mdx';
 
 <PublicPreviewBannerPartial />
 
 #### Step 1: Update Nitro on nodes and validators
 
-Refer to the release notes supporting the ArbOS version that you're upgrading to. You'll find a specific version of the Nitro software stack specified. For example, if your upgrade targets ArbOS 11, you'd use Nitro `v2.2.2` (Docker image: `offchainlabs/nitro-node:v2.2.2-8f33fea`) or higher. This is the version of the Nitro stack that needs to be running on each of your Orbit chain's nodes.
+Refer to the [requirements for the targeted ArbOS release](../../node-running/reference/arbos-software-releases/overview.mdx) to identify the specific [Nitro release](https://github.com/OffchainLabs/nitro/releases/) that supports the ArbOS version that you're upgrading to. For example, if your upgrade targets ArbOS 20, you'd use Nitro `v2.3.1` (Docker image: `offchainlabs/nitro-node:v2.3.1-26fad6f`) or higher. This is the version of the Nitro stack that needs to be running on each of your Orbit chain's nodes. A list of [all Nitro releases can be found on Github](https://github.com/OffchainLabs/nitro/releases).
 
 Begin by upgrading your validator node(s) to the specified Nitro version, then update each remaining Orbit node to match this version.
 
-Note that this update _must occur_ before the deadline established for the target ArbOS upgrade. Refer to the timestamp in the ArbOS upgrade schedule for a precise deadline.
+Note that upgrading your node version _must occur_ before the deadline established for the target ArbOS upgrade. Refer to the timestamp in the ArbOS upgrade schedule for a precise deadline.
+
+#### Step 2: Upgrade the Wasm module root & your chain's Nitro contracts
+While every ArbOS upgrade will require an update to the Wasm module root, not every ArbOS upgrade will require an upgrade to the chain's `nitro-contracts` version. 
+If necessary, as defined in the release notes for each ArbOS release, you may need to deploy new versions of Nitro contracts to the parent chain of your Orbit chain. These contracts include the rollup logic, fraud proof contracts, and interfaces for interacting with Nitro precompiles. 
```

Comment:
Idk if we need to make this clear here, but it might be updates to a small subset of the contracts, not necessarily all or nothing

Reply:
Agree with Harry there. As an example, the new orbit-actions upgrade only upgrades a subset of the contracts 1.2.1, leaving the rest on 1.1.1 .
In the list of what the nitro-contracts include, I'd add something about bridging logic too (although not sure if you see bridging as a subset of one of those).

Reply:
```suggestion
If necessary, as defined in the release notes for each ArbOS release, you may need to deploy new versions of some (or all) of the Nitro contracts to the parent chain of your Orbit chain. These contracts include the rollup logic, bridging logic, fraud proof contracts, and interfaces for interacting with Nitro precompiles. 
```